soundata/jams_utils.py

```diff
@@ -76,19 +78,57 @@ def jams_converter(
 
     # soundata tags
     if tags is not None:
-        if not isinstance(tags, annotations.Tags):
-            raise TypeError("tags should be of type annotations.Tags")
-        jam.annotations.append(tags_to_jams(tags, duration=jam.file_metadata.duration))
+        if isinstance(tags, annotations.Tags):
+            jam.annotations.append(
+                tags_to_jams(tags, duration=jam.file_metadata.duration)
+            )
+        elif isinstance(tags, annotations.MultiAnnotator):
+            jam.annotations.extend(
+                multiannotator_to_jams(tags, tags_w_annotator_to_jams)
+            )
+        else:
+            raise TypeError(
+                "tags should be of type annotations.Tags or annotations.MultiAnnotator"
+            )
 
     # soundata events
     if events is not None:
-        if not isinstance(events, annotations.Events):
-            raise TypeError("events should be of type annotations.Events")
-        jam.annotations.append(events_to_jams(events))
+        if isinstance(events, annotations.Events):
+            jam.annotations.append(events_to_jams(events))
+        elif isinstance(events, annotations.MultiAnnotator):
+            jam.annotations.extend(
+                multiannotator_to_jams(events, events_w_annotator_to_jams)
```

Comment:
my suggestion: the only difference between `events_to_jams` and `events_w_annotator_to_jams` is that the latter is expecting an `annotator` argument. To simplify code, I would add a `annotator=None` to the signature of both `events_to_jams` and `tags_to_jams`, and then call `multiannotator_to_jams(events, events_to_jams)` and same with events. Then we don't need the `xxx_w_annotator_to_jams`. Your tests shouldn't change. What do you think?

Reply:
Yep this is a great suggestion! I will merge the functions. I was avoiding modifying existing methods at first haha.